### ISSUE TITLE
Trouble installing openFrameworks with mingw64

### ISSUE THREAD

**Issue by misaelmercado1** [msys2/MINGW-packages#20568]
### Description / Steps to reproduce the issue

Downloaded msys , ran the mingw64 shell , ran the [pacman -Syu --noconfirm --needed] command two times , then ran these two commands to install open frameworks [cd your_oF_directory/scripts/msys2] [./install_dependencies.sh --noconfirm] then i tried compiling the openframeworks with cd your_oF_directory/libs/openFrameworksCompiled/project and make but when i make it returns this error: Package glm was not found in the pkg-config search path.
Perhaps you should add the directory containing `glm.pc'
to the PKG_CONFIG_PATH environment variable
Package 'glm' not found
/bin/sh: line 1: return: can only `return' from a function or sourced script
Package glm was not found in the pkg-config search path.
Perhaps you should add the directory containing `glm.pc'
to the PKG_CONFIG_PATH environment variable
Package 'glm' not found
/bin/sh: line 1: return: can only `return' from a function or sourced script
makefileCommon/config.shared.mk:311: *** couldn't find glm pkg-config package or it's dependencies, did you run the latest install_dependencies.sh?.

### Expected behavior

I expected it to compile the libraries of openframworks

### Actual behavior

Not working and returns an error.

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

MINGW64_NT-10.0-22631

### MINGW environments affected

- [X] MINGW64
- [ ] MINGW32
- [ ] UCRT64
- [ ] CLANG64
- [ ] CLANG32
- [ ] CLANGARM64

**Comment by Kreijstal** [comment]
did you install glm?
pacman -S $MINGW_PACKAGE_PREFIX-glm

**Comment by misaelmercado1** [comment]
yeah , i tried reinstalling and it says it is already up to date and reisntalls it.

**Comment by Biswa96** [comment]
Could you provide the link of install_dependencies.sh file?
